test/v1/controllers/requestParsers/validators/AmendPeriodicValidatorSpec.scala

```diff
@@ -16,3188 +16,1289 @@
+                |}
+                |""".stripMargin
+            )
+          )) shouldBe List(ValueFormatError.copy(paths = Some(Seq("/periodAllowableExpenses/depreciationAllowable"))))
+      }
+
+      "/periodAllowableExpenses/otherExpensesAllowable is invalid" in {
+        validator.validate(
+          AmendPeriodicRawData(
+            validNino,
+            validBusinessId,
+            validPeriodId,
+            Json.parse(
+              """
+                |{
+                |    "periodIncome": {
+                |        "turnover": 1000.99,
+                |        "other": 1000.99
+                |    },
+                |    "periodAllowableExpenses": {
+                |        "costOfGoodsAllowable": 1000.99,
+                |        "paymentsToSubcontractorsAllowable": 1000.99,
+                |        "wagesAndStaffCostsAllowable": 1000.99,
+                |        "carVanTravelExpensesAllowable": 1000.99,
+                |        "premisesRunningCostsAllowable": 1000.99,
+                |        "maintenanceCostsAllowable": 1000.99,
+                |        "adminCostsAllowable": 1000.99,
+                |        "businessEntertainmentCostsAllowable": 1000.99,
+                |        "advertisingCostsAllowable": 1000.99,
+                |        "interestOnBankOtherLoansAllowable": 1000.99,
+                |        "financeChargesAllowable": 1000.99,
+                |        "irrecoverableDebtsAllowable": 1000.99,
+                |        "professionalFeesAllowable": 1000.99,
+                |        "depreciationAllowable": 1000.99,
+                |        "otherExpensesAllowable": -1000.99
+                |    },
+                |    "periodDisallowableExpenses": {
+                |        "costOfGoodsDisallowable": 1000.99,
+                |        "paymentsToSubcontractorsDisallowable": 1000.99,
+                |        "wagesAndStaffCostsDisallowable": 1000.99,
+                |        "carVanTravelExpensesDisallowable": 1000.99,
+                |        "premisesRunningCostsDisallowable": 1000.99,
+                |        "maintenanceCostsDisallowable": 1000.99,
+                |        "adminCostsDisallowable": 1000.99,
+                |        "businessEntertainmentCostsDisallowable": 1000.99,
+                |        "advertisingCostsDisallowable": 1000.99,
+                |        "interestOnBankOtherLoansDisallowable": 1000.99,
+                |        "financeChargesDisallowable": 1000.99,
+                |        "irrecoverableDebtsDisallowable": 1000.99,
+                |        "professionalFeesDisallowable": 1000.99,
+                |        "depreciationDisallowable": 1000.99,
+                |        "otherExpensesDisallowable": 1000.99
+                |    }
+                |}
+                |""".stripMargin
+            )
+          )) shouldBe List(ValueFormatError.copy(paths = Some(Seq("/periodAllowableExpenses/otherExpensesAllowable"))))
+      }
```

Comment:
There don't seem to be any specific tests for `ValueFormatError` for `periodDisallowableExpenses` values.

**Note** that you can often save yourself copying swathes of json (with just one field having a bad value) by mixing the JsonErrorValidators trait into your spec class and replacing the chunk of JSON (`Json.parse("...etc etc etc...")`) with for example 
```
requestBodyJson.update("/periodAllowableExpenses/otherExpensesAllowable", JsNumber(123.123))
```